web/src/main/webapp/WEB-INF/data/data/formatter/xslt/render-layout.xsl

```diff
@@ -21,7 +21,7 @@
   <xsl:template mode="getMetadataTitle" match="*"/>
   <xsl:template mode="getMetadataAbstract" match="*"/>
   <xsl:template mode="getMetadataHierarchyLevel" match="*"/>
-  <xsl:template mode="getOverviews" match="*"/>
```

Comment:
I think you should not drop that template, because if you do that then a call to that mode for a record which does not implement it (eg. iso19110) it will not match anything.

Reply:
+1